elasticsearch/bulk/bulk.go

```diff
@@ -180,11 +185,47 @@ func (b *Bulk) flushMessages() error {
 func (b *Bulk) bulkRequest() error {
 	startedTime := time.Now()
 	reader := bytes.NewReader(b.batch)
-	_, err := b.esClient.Bulk(reader)
+	resp, err := b.esClient.Bulk(reader)
 	b.metric.BulkRequestProcessLatencyMs = time.Since(startedTime).Milliseconds()
 	if err != nil {
 		return err
 	}
+	err = b.handleResp(resp)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (b *Bulk) handleResp(resp *esapi.Response) error {
+	if resp == nil {
+		return fmt.Errorf("ERROR occurred when bulk request. Error: Response is nil")
+	}
+	if resp.IsError() {
+		return fmt.Errorf("ERROR occurred when bulk request. Error:%s", resp.String())
+	}
+	bodyBuffer := new(bytes.Buffer)
```

Comment:
this is not very necessary but we can rename bodyBuffer and bodyInterface "rb(responseBody), b(body)" according to this example https://github.com/golang/go/blob/7b325ba27d8a5a90d16793267523ebf52a18bf65/src/internal/pkgbits/encoder.go#L167

Reply:
I renamed it.